src/rgw/rgw_op.h

```diff
@@ -234,6 +241,11 @@ class RGWOp : public DoutPrefixProvider {
     this->store = store;
     this->s = s;
     this->dialect_handler = dialect_handler;
+    #ifdef HAVE_JAEGER
+    const auto trace_name = std::string(name()) + " " + s->trans_id;
+    this->trace = rgw_tracer->start_trace(trace_name);
```

Comment:
we'll want to avoid string formatting like this to minimize tracing overhead

`s->trans_id` probably makes more sense as a tag than a span name, and it looks like you can pass that directly to `SetTag()` as an integer, instead of formatting it as a string

the trace name itself should probably just be `"request"`, since we're already adding a tag with the RGWOp's `name()`

Reply:
in the flow, each trace is a "request" - and what uniquely identifies the request is the transaction id. why not use it as the name of the trace?
IMO, using transaction-id as a tag makes less sense since tags are mainly useful for searches

Reply:
ok, i'm not sure exactly how this effects what we see in the gui. but not all traces will come from client requests; others may come from multisite sync, lifecycle, gc, notifications, etc. if we have several different 'types' of traces, is it better to represent that in the name, or in a separate tag like "type=request"?

Reply:
ok. so lets add a "type" tag, that would be "request" for now
still think it is nice to have "request-<transaction id>" in the name, so it is clear these are distinct spans

Reply:
I added the type tag.